### PR TITLE
KMS: Make the encrypt/decrypt samples consistent with the other languages

### PR DESCRIPTION
**Pull request by russ-**
In particular, this change:
- changes to use a consistent "_id" suffix for resource name components
- uses "data" instead of "text", since encryption is not restricted to text
- substitutes "ciphertext" for "encrypted {text, data}"
- cases "cryptoKey" and "keyRing" consistently with the API

Tracking bug: http://b/64758639

### CODE REVIEW

File: kms/api-client/snippets.py [hunk 1]
```diff
@@ -61,70 +61,70 @@ def create_cryptokey(project_id, location, keyring, cryptokey):
...
-    # Read text from the input file.
+    # Read data from the input file.
     with io.open(plaintext_file_name, 'rb') as plaintext_file:
         plaintext = plaintext_file.read()
 
-    # Use the KMS API to encrypt the text.
-    cryptokeys = kms_client.projects().locations().keyRings().cryptoKeys()
-    request = cryptokeys.encrypt(
+    # Use the KMS API to encrypt the data.
+    cryptoKeys = kms_client.projects().locations().keyRings().cryptoKeys()
```
Comment (jonparrott) [inline kms/api-client/snippets.py]: our Python style guide is lower snake case for variable names. I understand the desire to match the API, but we've stuck to using lower snake case consistency for variables regardless of the API.

File: kms/api-client/snippets.py [hunk 2]
```diff
@@ -61,70 +61,70 @@ def create_cryptokey(project_id, location, keyring, cryptokey):
 
 
 # [START kms_encrypt]
-def encrypt(project_id, location, keyring, cryptokey, plaintext_file_name,
-            encrypted_file_name):
-    """Encrypts data from a plaintext_file_name using the provided CryptoKey
-    and saves it to an encrypted_file_name so it can only be recovered with a
-    call to decrypt."""
+def encrypt(project_id, location_id, key_ring_id, crypto_key_id,
```
Comment (jonparrott) [inline kms/api-client/snippets.py]: Since you are changing variable names, can you update the other samples in this file to match?

Comment (russ-) [inline kms/api-client/snippets.py]: Good catch, all fixed now.

Comment (russ-) [inline kms/api-client/snippets.py]: Good catch. Fixed.

File: kms/api-client/snippets.py (hunk 1 above)
Comment (russ-) [inline kms/api-client/snippets.py]: Fixed. Sorry, I didn't intend to go against the style guide, just an oversight on my part.

**Pull request merged**